Loader/ItemLoader.cs

```diff
@@ -161,7 +174,7 @@ public List<ItemIndexEntry> Load()
 				EntityClassDefinition magazine = null;
 				if (!String.IsNullOrEmpty(entity.Components?.SCItemWeaponComponentParams?.ammoContainerRecord))
 				{
-					magazine = ClassParser<EntityClassDefinition>.ClassByRefCache.GetValueOrDefault(entity.Components?.SCItemWeaponComponentParams.ammoContainerRecord);
+					magazine = ClassParser<EntityClassDefinition>.ClassByRefCache.GetValueOrDefault(entity.Components.SCItemWeaponComponentParams.ammoContainerRecord);
```

Comment:
There used to be certain items that didn't have fully fleshed out data, so it might be worth keeping this

Reply:
I removed the "?" because the code test just before if ammoContainerRecord is null or not.